src/test/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSinkConfigBuilder.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.gcp.gcs.sink;
+
+public class GCSBatchSinkConfigBuilder {
```

Comment:
our normal pattern is to put the Builder within the class that it builds. GCSBatchSinkConfig.Builder